core/invoker/src/main/scala/org/apache/openwhisk/core/invoker/Invoker.scala

```diff
@@ -156,25 +156,50 @@ object Invoker {
           .isFailure) {
       abort(s"failure during msgProvider.ensureTopic for topic $topicName")
     }
+
     val producer = msgProvider.getProducer(config, Some(ActivationEntityLimit.MAX_ACTIVATION_LIMIT))
     val invoker = try {
-      new InvokerReactive(config, invokerInstance, producer, poolConfig)
+      SpiLoader.get[InvokerProvider].instance(config, invokerInstance, producer, poolConfig, limitConfig)
     } catch {
       case e: Exception => abort(s"Failed to initialize reactive invoker: ${e.getMessage}")
     }
 
-    Scheduler.scheduleWaitAtMost(1.seconds)(() => {
```

Comment:
In some invoker implementations, health checking may not be performed via Kafka.
So I moved this to `InvokerReactive`.